src/manage-apps/manage-repos-terminal.md

```diff
@@ -0,0 +1,105 @@
+# Managing Repositories with the Terminal
+
+Use the `apt-manage` tool to add and remove Debian repositories in the `/etc/apt/sources.list.d/` directory. Use the `flatpak` tool to add and remove Flatpak repositories from the `~/.local/share/flatpak/repo/` directory.
+
+## Using the apt-manage Command
+
+The `apt-manage` command is unique to Pop!\_OS and comes from `repolib` the same library that Repoman is built upon. All options for the `apt-manage` command can be viewed by opening a Terminal and entering:
+
+```bash
+apt-manage -h
+```
+
+### Add a Repository Using apt-manage
+
+The `add` option will create an entry in the text file located in `/etc/apt/sources.list.d/` (Debian repositories). Enter this command and enter your password when prompted.
```

Comment:
I'm unclear what `(Debian repositories)` is doing in that first sentence.

Reply:
This sentence originally mentioned directories for both apt and flatpak repos.